src/components/Contacts/ContactDetails/ContactDontationsTab/ContactDonationsList/ContactDonationsList.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Box,
+  Button,
+  styled,
+  Table,
+  TableCell,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import { DateTime } from 'luxon';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { currencyFormat } from '../../../../../lib/intlFormat/intlFormat';
+import { useContactDonationsListQuery } from './ContactDonationsList.generated';
+
+interface ContactDonationsListProp {
+  accountListId: string;
+  contactId: string;
+}
+
+const LoadMoreButton = styled(Button)(({ theme }) => ({
+  margin: theme.spacing(1),
+}));
+
+const DonationLoadingPlaceHolder = styled(Skeleton)(({ theme }) => ({
+  width: '100%',
+  height: '24px',
+  margin: theme.spacing(2, 0),
+}));
+
+export const ContactDonationsList: React.FC<ContactDonationsListProp> = ({
+  accountListId,
+  contactId,
+}) => {
+  const [range, setRange] = useState(10);
+
+  const {
+    data,
+    loading,
+    fetchMore,
+    networkStatus,
+  } = useContactDonationsListQuery({
+    variables: {
+      accountListId: accountListId,
+      contactId: contactId,
+    },
+    notifyOnNetworkStatusChange: true,
+  });
+
+  const { t } = useTranslation();
+
+  return (
+    <Box>
+      {loading || networkStatus === 3 ? (
```

Comment:
If `notifyOnNetworkStatusChange` is set globally:
```suggestion
      {loading ? (
```